tests/bbsSignature/sign.bbsSignature.spec.ts

```diff
@@ -25,25 +25,37 @@ import {
 import { stringToBytes } from "../utilities";
 
 describe("bbsSignature", () => {
-  const blsKeyPair = generateBls12381G2KeyPair();
+  let blsKeyPair: BlsKeyPair;
+
+  beforeAll(async () => {
```

Comment:
any reason not to use beforeEach here instead of beforeAll to prevent keys from being reused between tests? It won't make much of a difference here just thinking about consistency with the isolation test principle.

Reply:
Yeap true we could opt to split this in future, unfortunately because of the performance of ASM.js doing that would add significant time to the CI pipeline with test execution